### PR TITLE
Bug 1970604:  Add IDP menu items are not translated

### PR DESCRIPTION
**Pull request by cyril-ui-developer**
/assign @rhamilto 
cc: @rebeccaalpert 

Screen Shot 2021-06-21 at 11 20 32 AM

### CODE REVIEW

File: frontend/public/components/cluster-settings/oauth.tsx
Comment (cyril-ui-developer) [inline frontend/public/components/cluster-settings/oauth.tsx:80]: @rhamilto @rebeccaalpert I am not sure why we were assigning `addIDPItems` object value here. I think it should be the key.

File: frontend/public/components/cluster-settings/oauth.tsx
Comment (cyril-ui-developer) [inline frontend/public/components/cluster-settings/oauth.tsx:58]: @rhamilto @rebeccaalpert i don't think we need the above any more.

**Comment by rhamilto** [comment]
LGTM but will wait for @rebeccaalpert to give the final approval as the resident i18n expert.  :)

File: frontend/public/components/cluster-settings/oauth.tsx
Comment (rebeccaalpert) [reply frontend/public/components/cluster-settings/oauth.tsx:58]: You're totally right; we don't need these anymore.

**Review by rebeccaalpert — APPROVED** [review]
Looks great to me. :) Thanks for fixing this.

**Pull request merged**
